ViewControllers/LoginViewController.swift

```diff
@@ -111,6 +112,21 @@ class LoginViewController: UIViewController {
             LocalStorage.instance.setUser(user)
             LocalStorage.instance.setDashXToken(dashXToken)
             LocalStorage.instance.setToken(response.token)
+            if let user = user {
+                let userDetails: NSDictionary = [
+                    UserAttributes.UID: user.idString as Any,
+                    UserAttributes.EMAIL: user.email as Any,
+                    UserAttributes.NAME: user.name,
+                    UserAttributes.FIRST_NAME: user.firstName as Any,
+                    UserAttributes.LAST_NAME: user.lastName as Any
+                ]
+                DashX.setIdentity(uid: user.idString)
+                do {
+                    try DashX.identify(withOptions: userDetails)
```

Comment:
Just keep setIdentity. No need to save registered user details via SDK as the backend API is already doing that